### ISSUE TITLE
REPAINT: Knowledge Transfer in Deep Actor-Critic Reinforcement Learning. (arXiv:2011.11827v1 [cs.LG])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#1376]
REPAINT: Knowledge Transfer in Deep Actor-Critic Reinforcement Learning. (arXiv:2011.11827v1 [cs.LG])<br>
https://ift.tt/2J79dH2<br>
<p>Accelerating the learning processes for complex tasks by leveraging previously learned tasks has been one of the most challenging problems in reinforcement learning, especially when the similarity between source and target tasks is low or unknown. In this work, we propose a REPresentation-And-INstance Transfer algorithm (REPAINT) for deep actor-critic reinforcement learning paradigm. In representation transfer, we adopt a kickstarted training method using a pre-trained teacher policy by introducing an auxiliary cross-entropy loss. In instance transfer, we develop a sampling approach, i.e., advantage-based experience replay, on transitions collected following the teacher policy, where only the samples with high advantage estimates are retained for policy update. We consider both learning an unseen target task by transferring from previously learned teacher tasks and learning a partially unseen task composed of multiple sub-tasks by transferring from a pre-learned teacher sub-task. In several benchmark experiments, REPAINT significantly reduces the total training time and improves the asymptotic performance compared to training with no prior knowledge and other baselines.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
